firefox-ios/Client/metrics.yaml

```diff
@@ -4870,6 +4870,49 @@ app_errors:
     notification_emails:
       - fx-ios-data-stewards@mozilla.com
     expires: never
+  crashed_last_launch:
+    type: event
+    description: |
+      Recorded when the previous session ended as a result of a crash
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-7558
```

Comment:
Please update these links to the github link

Reply:
Updated 👍